code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -141,6 +141,8 @@
 	var/bomb_strength = 0
 	///Delay between firing the bombard ability for boilers
 	var/bomb_delay = 0
+	///Used to reduce cooldown for the boiler
+	var/ammo_multiplier = 0
```

Comment:
weird to have this as a var IMO, but whatever

Reply:
Dunno what else you would want this to be, really